package.json

```diff
@@ -122,6 +122,7 @@
     "unist-builder": "^3.0.1",
     "unist-util-visit": "^4.1.2",
     "vscode-ripgrep": "^1.13.2",
+    "web-features": "^0.3.0",
```

Comment:
⚠️  This package does not have a license yet.

Reply:
It does now :)